changelog.md

```diff
@@ -1,8 +1,10 @@
 # Backpack changelog
 
-## UNRELEASED
+## UNRELEASED - New BpkThemeProvider component
```

Comment:
We don't tend to add titles until release point! I guess it's not really a problem but it just looks weird!

Reply:
Yeah i'd add that you can only really provide a title after you have released - who know what is going to make that particular release until you actually do it?

Reply:
Yeah, I looked at it and I thought actually I'll write it now, because I figured the `unreleased' part was in reference to the datetime. Happy to remove if people prefer, of course.